### DISCUSSION TITLE
DAPP连接tron钱包的问题

### DISCUSSION THREAD

**Discussion by s1mple1122** [consenlabs/webview#57] [Q&A] [answered]
在deeplink页面,我找到了wallet connect 的方式,但是这个方式是否写得模糊,有没有具体得参数之类的 
bridge? wc? key?这些.钱包是否都没有说明
我这边需要开发一个dapp,使用tron钱包

**Comment by xwartz** [comment, marked as answer]
我们目前不支持 tron 钱包的 wallet connect

**Reply by s1mple1122** [reply]
那我该如何使用tron钱包呢

**Reply by xwartz** [reply]
可以直接使用 [tronweb](https://github.com/tronprotocol/tronweb)

**Reply by s1mple1122** [reply]
tronweb不是与tron网络交互的吗?不是很明白你的意思

**Reply by s1mple1122** [reply]
我是想连接imtoken的tron钱包, 然后转载usdt之类的操作

**Reply by xwartz** [reply]
和以太坊类似的，imToken 注入了拦截代码，会拦截 tronweb 的签名方法

**Reply by s1mple1122** [reply]
或者我们在简单一点,我想获取用户的tron钱包地址,有没有此类方法

**Reply by xwartz** [reply]
`window.tronWeb.address`

**Reply by s1mple1122** [reply]
这是哪个js包,麻烦你告诉我下,或者文档位置

**Reply by xwartz** [reply]
就是我上面说的 tronweb

**Reply by s1mple1122** [reply]
好的,多谢
